infrastructure/docker/php/docker-entrypoint.sh

```diff
@@ -60,12 +60,7 @@ if [ "$1" = 'php-fpm' ] || [ "$1" = 'php' ] || [ "$1" = 'bin/console' ]; then
 			bin/console doctrine:migrations:migrate --no-interaction
 		fi
 	fi
-  if [ "$APP_ENV" = 'prod' ]; then
-      setfacl -R -m u:www-data:rwX -m u:"$(whoami)":rwX var
-      setfacl -dR -m u:www-data:rwX -m u:"$(whoami)":rwX var
-  else
-      chmod -R 755 var
-  fi
+  chmod -R 755 var
```

Comment:
Let's use setfacl if it's supported
In other cases we can use chmod

For example:
```
if command -v setfacl >/dev/null 2>&1; then
    # Check if ACLs are supported by trying to apply an ACL
    if setfacl -m u::rwx "$TARGET_DIR" 2>/dev/null; then
        echo "ACLs are supported. Setting ACLs..."
        setfacl -R -m u:www-data:rwX -m u:"$(whoami)":rwX "$TARGET_DIR"
    else
        echo "ACLs are not supported. Falling back to chmod..."
        chmod 755 -R "$TARGET_DIR"
    fi
else
    echo "setfacl command not found. Using chmod..."
    chmod 755 -R "$TARGET_DIR"
fi
```